jest.config.js

```diff
@@ -5,12 +5,12 @@ module.exports = {
   modulePathIgnorePatterns: ['<rootDir>/examples/'],
   setupFilesAfterEnv: ['<rootDir>/test/jest-setup.ts'],
   moduleNameMapper: {
-    '^swr$': '<rootDir>/core/src/index.ts',
-    '^swr/infinite$': '<rootDir>/infinite/src/index.ts',
-    '^swr/immutable$': '<rootDir>/immutable/src/index.ts',
-    '^swr/subscription$': '<rootDir>/subscription/src/index.ts',
-    '^swr/mutation$': '<rootDir>/mutation/src/index.ts',
-    '^swr/_internal$': '<rootDir>/_internal/src/index.ts'
+    '^swr$': '<rootDir>/src/index.ts',
```

Comment:
This file doesn't exist.

```suggestion
    '^swr$': '<rootDir>/src/core/index.ts',
```